src/env.ts

```diff
@@ -373,6 +373,7 @@ export const env = {
       'https://api.ud-sandbox.com/metadata',
   },
   PUBSUB: {
+    ENABLE: process.env.PUBSUB_ENABLE || true,
```

Comment:
```suggestion
    ENABLE: process.env.DOMAIN_ACTIONS_PUBSUB_ENABLE || false,
```
can we wrap this into FEATURE_FLAGS
I also have similar flag for image processing https://github.com/unstoppabledomains/resolution-service/pull/393/files